app/src/main/java/fr/neamar/kiss/dataprovider/AppProvider.java

```diff
@@ -113,6 +117,70 @@ public void reload() {
         this.initialize(new LoadAppPojos(this));
     }
 
+	/**
+	 * @param query    The string to search for
+	 * @param searcher The receiver of results
+	 */
+
+	@Override
+	public void requestResults( String query, Searcher searcher )
+	{
+		String queryNormalized = StringNormalizer.normalize( query );
+
+		FuzzyScore           fuzzyScore = new FuzzyScore();
+		FuzzyScore.MatchInfo matchInfo  = new FuzzyScore.MatchInfo();
+
+		for( AppPojo pojo : pojos )
+		{
+			pojo.displayName = pojo.name;
```

Comment:
Is it worth setting the displayName even if it's not a match?
Same for displayTags